website/content/docs/job-specification/transparent_proxy.mdx

```diff
@@ -0,0 +1,179 @@
+---
+layout: docs
+page_title: transparent_proxy Block - Job Specification
+description: |-
+  The "transparent_proxy" block allows specifying options for configuring Envoy
+  in Consul Connect transparent proxy mode.
+---
+
+# `transparent_proxy` Block
+
+<Placement
+  groups={[
+    'job',
+    'group',
+    'service',
+    'connect',
+    'sidecar_service',
+    'proxy',
+    'transparent_proxy',
+  ]}
+/>
+
+The `transparent_proxy` block configures the Envoy sidecar proxy to act as a
+Consul Connect [transparent proxy][tproxy]. This simplifies the configuration of
+Consul Connect by eliminating the need to configure [`upstreams`][] blocks in
+Nomad. Instead, the Envoy proxy will takes its configuration entirely from
+Consul [service intentions][]. When transparent proxy is enabled:
+
+* Nomad will invoke the [`consul-cni`][] CNI plugin to configure `iptables` rules
```

Comment:
this section of internal nuts-and-bolts is really interesting, and good for cluster operators to learn, but I wonder if it belongs here in the `transparent_proxy{}` block usage doc, which is mostly geared towards job authors, if I'm not mistaken?

usually when I come here, I mainly want to see the params and examples, then link(s) elsewhere for more context if needed for the feature.

Reply:
Yeah, this needs to exist somewhere, but maybe not here. I'll move it over to [here](https://github.com/hashicorp/nomad/pull/20251/files#diff-63fd8339939096531fd9c05a483e95c823c771f4224e33325a2ddf02b9f7c6deR137) (so there'll be a dead link in this PR until that's merged and this is rebased)